update.go

```diff
@@ -0,0 +1,191 @@
+// +build linux
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/codegangsta/cli"
+)
+
+type updateRessources struct {
```

Comment:
That struct contains a lot more fields (e.g. Devices array, OOM, Pids, Network, etc..)